recipes/aiozmq/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "aiozmq" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b41d589a715acc0d42b525ef7084bbcdc03c986ef5a90e212ee7f086125cf6c0
+
+build:
+  number: 0
```

This can probably be made a [`noarch` package](https://cf-docs-restructure.readthedocs.io/en/docs_restructure/maintainer/knowledge_base.html#noarch-builds).
```suggestion
  noarch: python
  number: 0
```